op-program/chainconfig/configs/placeholder.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this? Can it just be `.gitkeep` or does the `go:embed` complain if it doesn't find anything?

Reply:
Yes, `go:embed` will complain and stop compiling if no file matches the pattern.